tests/LSP1UniversalReceiver/LSP1UniversalReceiverDelegateUP.behaviour.ts

```diff
@@ -1961,7 +1961,7 @@ export const shouldInitializeLikeLSP1Delegate = (
     it("should have registered the LSP1Delegate interface", async () => {
```

Comment:
Don't forget to edit the test title 🙂 

```suggestion
    it("should have registered the LSP1 interface", async () => {
```